src/leap/mail/imap/mailbox.py

```diff
@@ -514,13 +514,14 @@ def _get_imap_msg(messages):
                 d_imapmsg = []
                 for msg in messages:
                     d_imapmsg.append(getimapmsg(msg))
-                return defer.gatherResults(d_imapmsg)
+                return defer.gatherResults(d_imapmsg, consumeErrors=True)
```

Comment:
```
    @param consumeErrors: (keyword param) a flag, defaulting to False,
    indicating that failures in any of the given L{Deferreds} should not be
    propagated to errbacks added to the individual L{Deferreds} after this
    L{gatherResults} invocation.  Any such errors in the individual
    L{Deferred}s will be converted to a callback result of C{None}.  This                                                                                                    
    is useful to prevent spurious 'Unhandled error in Deferred' messages
    from being logged.  This parameter is available since 11.1.0.
```